legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/sqlQueryToString/SQLQueryToString.pure

```diff
@@ -1060,7 +1072,8 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::getDy
                                                  choice(DatabaseType.Postgres,     $allStates,            ^ToSql(format='atan2(%s,%s)')),
                                                  choice(DatabaseType.Snowflake,    $allStates,            ^ToSql(format='atan2(%s,%s)')),
                                                  choice(DatabaseType.Presto,       $allStates,            ^ToSql(format='atan2(%s,%s)')),
-                                                 choice(DatabaseType.BigQuery,       $allStates,          ^ToSql(format='atan2(%s,%s)')),                                                 
+                                                 choice(DatabaseType.BigQuery,       $allStates,          ^ToSql(format='atan2(%s,%s)')),
+                                                 choice(DatabaseType.RedShift, $allStates, ^ToSql(format='atan2(%s,%s)')),

```

Comment:
let's follow indentation practice .
NitPick -> $allStates does not seem to be indented correctly